src/2048-expert/2048-expert.css

```diff
@@ -0,0 +1,80 @@
+/*
+Author: Zihan Zhong
```

Comment:
All the files in this repository are open source and shared across all collaborators so I don't think we need to mention the author of each file.